src/media/media.ts

```diff
@@ -701,19 +701,18 @@ export class SdkMedia extends (EventEmitter as { new(): StrictEventEmitter<Event
   private getStandardConstraints (options: IMediaRequestOptions): MediaStreamConstraints {
     const constraints: any = {
       video: {},
-      audio: {}
+      audio: {
+        googAudioMirroring: false,
```

Comment:
extra constraints should be ignored by browsers so the `goog` options should be fine.

Reply:
I wonder if we should test this for hardware setups that maybe don't support `autoGainControl` and/or `echoCancellation`? I'm not sure if that's possible, but I'm thinking back to the FF video resolution where gUM threw an error because the hardware setup wasn't allowing the requested constraints. 

The "hailmary" gUM retry will use the defaults returned from here. So if there is ever a chance these settings could throw, there won't be a way around them.

Reply:
So the reason I didn't do any extra here was because of this: https://developer.mozilla.org/en-US/docs/Web/API/Media_Streams_API/Constraints

```
If needed, call MediaDevices.getSupportedConstraints() to get the list of supported constraints, which tells you what constrainable properties the browser knows about. This isn't always necessary, since any that aren't known will be ignored when you specify them—but if you have any that you can't get by without, you can start by checking to be sure they're on the list.
```

I has just planned on leaving it and revisiting if needed. I think we should be fine here though.

Reply:
Looks like we should be good then 👍